test/unit/loggingManagerTests.js

```diff
@@ -89,40 +99,47 @@ describe('LoggingManager', function () {
         )
       })
 
-      it('should not run checkLogLevel', function () {
+      it('should not run getTracingEndTime', function () {
```

Comment:
This seems to be an unrelated change? The test-case is actually checking the `checkLogLevel` stub, so the previous title fits better here.